### PR TITLE
Jetpack: Replace HEREDOC syntax with strings

### PR DESCRIPTION
**Pull request by bindlegirl**
Fixes https://github.com/Automattic/vulcan/issues/336
## Proposed changes:

* Replaced HEREDOC syntax with multiline strings.

### Other information:

- [ ] Have you written new tests for your changes, if applicable?
- [ ] Have you checked the E2E test CI results, and verified that your changes do not break them?
- [ ] Have you tested your changes on WordPress.com, if applicable (if so, you'll see a generated comment below with a script to run)?

## Testing instructions:

* Does it make sense?
* Does it break anything (logo or module syncing)?

### CODE REVIEW

**Review by coder-karen — COMMENTED** [review]
I both checked the JP logo on a test site and changed modules then scheduled a checksum and fix with the Jetpack debugger, and that had no issues, but I'm not so familiar with sync so perhaps there is a better way to do that?

Also, you should be able to remove the disabled sniff `PEAR.Functions.FunctionCallSignature.Indent` just by indenting the SQL in those two blocks by three tabs :) (unless of course that affects functionality somehow, I can't see how it would from similar searches)

**Comment by fgiannar** [comment]
> I both checked the JP logo on a test site and changed modules then scheduled a checksum and fix with the Jetpack debugger, and that had no issues, but I'm not so familiar with sync so perhaps there is a better way to do that?

The changes are related with the Full Sync behaviour, so we can test this when the PR is ready for another review by triggering a Full Sync of all modules via the JB Debugger's Full Sync section :)

**Comment by bindlegirl** [comment]
Thanks for checking it out @coder-karen ! I indented the SQL code as you suggested and tested the Full Sync as @fgiannar suggested.

File: projects/packages/sync/src/modules/class-module.php [hunk 1]
```diff
@@ -317,17 +317,17 @@ public function get_next_chunk( $config, $status, $chunk_size ) {
 	 */
 	public function get_last_item( $config ) {
 		global $wpdb;
-		// phpcs:disable WordPress.DB.PreparedSQL.InterpolatedNotPrepared,WordPress.DB.DirectDatabaseQuery.NoCaching,WordPress.DB.DirectDatabaseQuery.DirectQuery
+		// phpcs:disable WordPress.DB.PreparedSQL.InterpolatedNotPrepared,WordPress.DB.DirectDatabaseQuery.NoCaching,WordPress.DB.DirectDatabaseQuery.DirectQuery,Generic.WhiteSpace.ScopeIndent.Incorrect
```
Comment (fgiannar) [inline projects/packages/sync/src/modules/class-module.php]: Do we still need this change now that the indent is fixed?

File: projects/packages/sync/src/modules/class-module.php [hunk 2]
```diff
@@ -317,17 +317,17 @@ public function get_next_chunk( $config, $status, $chunk_size ) {
...
+			"
+			SELECT {$this->id_field()}
+			FROM {$wpdb->{$this->table_name()}}
+			WHERE {$this->get_where_sql( $config )}
+			ORDER BY {$this->id_field()}
+			LIMIT 1
+			"
 		);
-		// phpcs:enable WordPress.DB.PreparedSQL.InterpolatedNotPrepared,WordPress.DB.DirectDatabaseQuery.NoCaching,WordPress.DB.DirectDatabaseQuery.DirectQuery
+		// phpcs:enable WordPress.DB.PreparedSQL.InterpolatedNotPrepared,WordPress.DB.DirectDatabaseQuery.NoCaching,WordPress.DB.DirectDatabaseQuery.DirectQuery,Generic.WhiteSpace.ScopeIndent.Incorrect
```
Comment (fgiannar) [inline projects/packages/sync/src/modules/class-module.php]: Same as https://github.com/Automattic/jetpack/pull/37396/files#r1606612166

File: projects/packages/sync/src/modules/class-module.php (hunk 1 above)
Comment (bindlegirl) [reply projects/packages/sync/src/modules/class-module.php]: Good catch. I removed one exception but missed a second one. Updated in [8a86004](https://github.com/Automattic/jetpack/pull/37396/commits/8a86004da67647f07580f5962185e1c354d4a949)

**Review by fgiannar — APPROVED** [review]
LGTM 🚢

**Pull request merged**